sys-block/sg3_utils/sg3_utils-1.42.recipe

```diff
@@ -0,0 +1,116 @@
+SUMMARY="A collection of utilities for interacting with the sg SCSI interface"
+DESCRIPTION="Provides utilities that send SCSI commands to devices. As well as \
```

Comment:
Thanks @alaviss for this excellent work!
sg3_utils, with your patch, builds perfectly on all architectures.
I'll try to test it on a computer I have which has a SATA DVD burner. I'll try to test a few of the sg_ utils to see if it can talk to that device.

I have 2 cosmetic suggestions for the recipe:
* It might be worth replacing the very first word in DESCRIPTION, "Provides", by "The sg3_utils package contains". This is because it is OK to mention the name of the package in this variable. (The resulting description will be exactly what the project's home page displays.)
* For SUMMARY, I would suggest:
```diff
-SUMMARY="A collection of utilities for interacting with the sg SCSI interface"
+SUMMARY="Utilities for interacting with the sg SCSI interface"
```
The current content is 68 chars, but I have never been able to answer this question: "Is the 70 chars limit for the whole line or only for the contents of the variable?" (Maybe I should run a survey :)

Reply:
Rewrapped, in case you would liek to copy/paste :)
```
SUMMARY="Utilities for interacting with the sg SCSI interface"
DESCRIPTION="The sg3_utils packages contains utilities that send SCSI commands \
to devices. As well as devices on transports traditionally associated with \
SCSI (e.g. Fibre Channel (FCP), Serial Attached SCSI (SAS) and the SCSI \
Parallel Interface(SPI)) many other devices use SCSI command sets. ATAPI \
cd/dvd drives and SATA disks that connect via a translation layer or a bridge \
device are examples of devices that use SCSI command sets."
```